src/components/organisms/PageHeader/PageHeader.tsx

```diff
@@ -234,23 +234,33 @@ const PageHeader = () => {
             <StyledLogo onClick={showStartupModal} src={MonokleKubeshopLogo} alt="Monokle" />
           </LogoCol>
           <SettingsCol span={12}>
-            <GitHubIconSpan>
-              <QuestionCircleOutlined size={24} onClick={openDocumentation} />
-            </GitHubIconSpan>
-            <StyledNotificationsBadge>
-              <span onClick={openDiscord}>
-                <img src={DiscordLogo} style={{height: '24px', cursor: 'pointer'}} />
-              </span>
-            </StyledNotificationsBadge>
-            <GitHubIconSpan>
-              <GithubOutlined size={24} onClick={openGitHub} />
-            </GitHubIconSpan>
-            <StyledSettingsBadge>
-              <StyledSettingsOutlined onClick={toggleSettingsDrawer} />
-            </StyledSettingsBadge>
-            <StyledNotificationsBadge>
-              <StyledBellOutlined onClick={toggleNotificationsDrawer} />
-            </StyledNotificationsBadge>
+            <Tooltip title="Documentation" placement="left">
```

Comment:
please move all tooltips to tooltips.ts and change as follows:
- "Open Monokle Documentation Website"
- "Open Kubeshop Discord Server"
- "Open Monokle Github Repository"
- "Open Settings"
- "Show latest notifications"
?

Reply:
Done